GravityZone Content Pack Contribution (#42789)
demisto/content: Packs/GravityZone/README.md

# GravityZone

Use the GravityZone integration to retrieve security events from GravityZone Cloud and manage incident response activities in Cortex XSOAR.

## What does this pack do?

This pack lets you:

- Retrieve security incidents and endpoint data from GravityZone Cloud.
- Perform incident response and remediation actions on managed endpoints.
- Mirror incident data between GravityZone Cloud and Cortex XSOAR.

This content pack includes:

- An integration with more than 10 commands for endpoint and incident management.

> - A classifier, as well as an incoming mapper to support incident data mirroring.

Reviewer: ```suggestion
- A classifier and an incoming mapper to support incident data mirroring.
```

# GravityZone

Use the GravityZone integration to retrieve security events from GravityZone Cloud and manage incident response activities in Cortex XSOAR.

## What does this pack do?

This pack lets you:

> 

Author: Fixed.
